lib/1.4/dml-builtins.dml

```diff
@@ -539,7 +539,8 @@ template _conf_attribute {
 
     param configuration             default "optional";
     param persistent                default false;
-    param internal                  default !defined documentation;
+    param internal                  default (!defined documentation &&
+                                             !defined desc);
```

Comment:
The style requires the binary operator after the line break, not before. Parentheses are optional, but seemingly encouraged, so you can keep those if you want to.

Reply:
I wholeheartedly agree! I have constantly been grudging internally about this ever since I joined in 2007, along with the c-style's indent level and `\n{`. I got used to it, and I always help reminding others to also obey, but I will never get used to it. A nice style is better than an ugly style, but an ugly style is better than an inconsistent mix of a nice style and an ugly style.